application/tests/helpers/api_helper_test.php

```diff
@@ -6,13 +6,21 @@ class APIHelperTest extends TestCase
     public function setUp() {
         $CI =& get_instance();
         $CI->load->helper('api');
+
+        // This tool is provided by sp1d3R in the HackerOne Bug Bounty program
+        // If it goes away, we'll need some other way to easily generate a redirect to an internal URL
+        $this->ssrfRedirectURL = "http://redir.xpoc.pro/127.0.0.1";
```

Comment:
I'm having trouble thinking of a way to generate the test case without relying on an external, untrusted service. I saw we were doing this in another test in BATS, though, so decided to just go with it for now so I could get this fix in.